slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/DbSlashingProtection.java

```diff
@@ -70,6 +78,24 @@ public DbSlashingProtection(
     this.signedBlocksDao = signedBlocksDao;
     this.signedAttestationsDao = signedAttestationsDao;
     this.registeredValidators = registeredValidators;
+    this.interchangeManager =
+        new InterchangeV5Manager(
+            jdbi,
+            validatorsDao,
+            signedBlocksDao,
+            signedAttestationsDao,
+            new ObjectMapper()
+                .registerModule(new InterchangeModule())
+                .configure(FLUSH_AFTER_WRITE_VALUE, true));
```

Comment:
Guessing this is needed because we are streaming the writes now?

Reply:
I think so ... we can manually flush, but this seems nicer.